docs/source/60_building_apps.md

```diff
@@ -256,6 +256,31 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s RegionJoin API
```

Comment:
I think this section is in the wrong doc, should it not be in `70_algorithms.md`?